server/migrations/20181029191911-create-user.js

```diff
@@ -7,7 +7,7 @@ export default {
       type: Sequelize.INTEGER
     },
     fullName: {
-      allowNull: false,
+      allowNull: true,
```

Comment:
Why are we allowing this to be null?

Reply:
Twitter does not return the first name and last name for users, just the display name and username. I had to make this field allow nulls to allow for creating new Twitter users.